### PR TITLE
Update web.config and project.json

### PR DESCRIPTION
**Pull request by JunTaoLuo**
- add `web.config` to `content`
- standardize contents of `web.config` to what other samples look like
- sort dependencies
- change indentation size

cc @Tratcher @davidfowl

### CODE REVIEW

File: HelloMvc/project.json [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "portable-net45+wp80+win8+wpa81+dnxcore50"
+      ]
     }
+  },
+  "tools": {
+    "dotnet-publish-iis": "1.0.0-*"
```
Comment (Tratcher) [inline HelloMvc/project.json]: Switch to the new tool name.

**Comment by Tratcher** [comment]
:shipit:

File: HelloMvc/project.json (hunk 1 above)
Comment (moozzyk) [inline HelloMvc/project.json]: I don't think it will work at all at this point. The package is not what we want yet because we have an oldish cli build.

Comment (Tratcher) [inline HelloMvc/project.json]: These samples are more about what we want to work, not what actually works yet.

Comment (moozzyk) [inline HelloMvc/project.json]: I don't know - people are trying them and have all sorts of issues...

Comment (Tratcher) [inline HelloMvc/project.json]: Yeah, but that's because nothing works at the moment... it's not just the tools.

Comment (JunTaoLuo) [inline HelloMvc/project.json]: Same goes for %APPLICATION_PATH% in web.config. It just serves as a placeholder to illustrate what should be there.

Comment (Tratcher) [inline HelloMvc/project.json]: VS will actually set those variables rather than modifying the file for IIS Express.

Comment (JunTaoLuo) [inline HelloMvc/project.json]: Eventually, I don't think it works yet.

**Pull request merged**